bindings/java/doxygen/artifact_catalog.dox

```diff
@@ -35,6 +35,23 @@ TSK_CARD_NUMBER (Credit card number)
 
 ---
 
+## TSK_ACTIVITY
+Activity on the system or from an application.  Example Usage is a mobile device being locked and unlocked.
+. 
+
+### REQUIRED ATTRIBUTES
+- TSK_DATETIME_START (When activity Started)
+    or 
+  TSK_DATETIME
+
+### OPTIONAL ATTRIBUTES
+- TSK_ACTIVITY_TYPE (Activity Type ie: On or Off)
```

Comment:
"ie" should be "i.e."

Reply:
Fixed that and a typo in one of the attribute names.